sdk/storage/azure-storage-blob/src/main/java/com/azure/android/storage/blob/models/ListBlobFlatSegmentHeaders.java

```diff
@@ -13,7 +13,7 @@
  * Defines headers for ListBlobFlatSegment operation.
  */
 @JacksonXmlRootElement(localName = "Container-ListBlobFlatSegment-Headers")
-public final class ContainerListBlobFlatSegmentHeaders {
+public final class ListBlobFlatSegmentHeaders {
```

Comment:
Naming: Is this going to be `ListBlobFlatSegmentHeaders`, `ListBlobFlatSegmentsHeaders` or `ListBlobsHeaders`?

Reply:
ideally, the name should be coming from swagger, we can always change the name through README transformations, lets do that as part of releasing storage, and ensure consistent with iOS.